packages/muelu/src/Graph/StructuredAggregation/MueLu_AggregationStructuredAlgorithm_def.hpp

```diff
@@ -50,7 +50,10 @@
 #include <Teuchos_Comm.hpp>
 #include <Teuchos_CommHelpers.hpp>
 
-#include <Xpetra_Vector.hpp>
+#include <Xpetra_Matrix.hpp>
```

Comment:
Remove this header as it relies on `Scalar` and switch to `xpetra/src/Map/Xpetra_MapFactory.hpp`